lib/index.js

```diff
@@ -137,40 +134,42 @@ function configure() {
   });
 }
 
+async function listenHttps() {
+  const logger = bedrock.loggers.get('app');
+  const port = bedrock.config.server.port;
+  return Promise.all(bedrock.config.server.bindAddr.map(async addr => {
+    logger.debug('starting HTTPS server on %s:%d', addr, port);
+    await bedrock.events.emit(
+      'bedrock-server.https.listen', {address: addr, port});
+    await new Promise(resolve => {
+      api.servers.https.listen(port, addr, () => resolve());
+    });
```

Comment:
Was just looking at the `listen` API which is weird: https://nodejs.org/docs/latest-v12.x/api/net.html#net_server_listen

Wondering happens to errors like EADDRINUSE

Is the callback being called with an error not a thing and therefore there's no need to be concerned about rejecting the Promise?

Since `listen` is a sync API by default, would it just be easier to use it that way?

is the additional arrow function to cast away any value sent to the callback?  Could it be:
```
await new Promise(resolve => api.servers.https.listen(port, addr, resolve);
```

Reply:
I think the error handling is just done via that 'error' event handler. That just throws the error. I've had it fail to start from that in the past, so I'm guessing it still works though I didn't test it with new code.

It uses that last callback as a 'listening' event listener.  Seems to work with that shorter syntax, so I'll change it.